deployer/src/config_creator.py

```diff
@@ -33,7 +33,14 @@ def extract_root_from_input(input_string):
         return domain.group() if domain else input_string
 
 
+def is_non_empty_list(is_non_empty_list):
+    # Check if the input is a non-empty list. Otherwise, it raises an exception
+    if not is_non_empty_list or len(is_non_empty_list) == 0:
+        raise Exception('{} iss not a non-empty list'.format(is_non_empty_list))
```

Comment:
Typo: "is" instead of "iss"
Let's also avoid double negations: "{} is not a non-empty list" => say instead "{} is None or empty"